src/main/java/org/mockito/internal/util/Primitives.java

```diff
@@ -60,6 +63,17 @@ public static boolean isAssignableFromWrapper(Class<?> valueClass, Class<?> refe
         return (T) PRIMITIVE_OR_WRAPPER_DEFAULT_VALUES.get(primitiveOrWrapperType);
     }
 
+    /**
+     * Returns the boxed default value for the lambda return type (handling primitive and primitive wrapper types).
+     *
+     * @param consumer the consumer function
+     * @param <T> the function's return type
+     * @return the boxed default value for the lambda return type (handling primitive and primitive wrapper types)
+     */
+    public static <T> T defaultValueForConsumerLambda(Consumer<T> consumer) {
+        Class<?>[] typeArgs = TypeResolver.resolveRawArguments(Consumer.class, consumer.getClass());
```

Comment:
@raphw How to get an `TypeDescription` for a Class-Object? I looked into the byte-buddy code but  don't get it.